### PR TITLE
[CIS-727] `ChatChannel` limits refinements

### PR DESCRIPTION
**Pull request by VojtaStavik**
This is a follow-up PR on #911 . It improves the naming and adds a limit for `ChatChannel.latestMesages`.

### CODE REVIEW

File: Sources/StreamChat/Database/DTOs/ChannelDTO.swift [hunk 1]
```diff
@@ -318,8 +322,7 @@ extension _ChatChannel {
             cooldownDuration: Int(dto.cooldownDuration),
             extraData: extraData,
 //            invitedMembers: [],
-            latestMessages: { fetchMessages(25) }, // TODO: make messagesLimit a param
-            lastMessage: fetchMessages(1).first,
+            latestMessages: { fetchMessages() }, // TODO: make messagesLimit a param
```
Comment (b-onc) [inline Sources/StreamChat/Database/DTOs/ChannelDTO.swift:325]: We have `latestMessagesLimit` now?

Comment (VojtaStavik) [reply Sources/StreamChat/Database/DTOs/ChannelDTO.swift:325]: Thanks!

**Review by b-onc — APPROVED** [review]
One question, rest LGTM

**Comment by nuno-vieira** [comment]
I've enabled this in the settings of the project:
image

So we now need to rebase main into the branch before merging. This will make sure to not break the main branch again.

**Pull request merged**